### ISSUE TITLE
Product Collection Item has adjacent links with the same destination, cuasing lower site scores on accessibility tests.

### ISSUE THREAD

**Issue by SEWEC** [adobe/aem-core-cif-components#541]
### Expected Behaviour
It seems that to meet accessibility standards the [Product Collection Item](https://github.com/adobe/aem-core-cif-components/blob/master/ui.apps/src/main/content/jcr_root/apps/core/cif/components/commerce/productcollection/v1/productcollection/item.html)  should be made with only one link covering both the title and image.
### Actual Behaviour
There are redundant links on the item image and item title.
#### Steps to Reproduce
Use the WAVE web accessibility evaluation tool(available as a [firefox/chrome extension](https://wave.webaim.org)) or Siteimprove accessibility tool on a page using the product collection component and notice the warning given for adjacent redundant links.
#### Platform and Version
AEM 6.5, CIF core 1.9
#### Logs taken while reproducing problem
Alerts
Redundant link
What It Means
Adjacent links go to the same URL.
Why It Matters
When adjacent links go to the same location (such as a linked product image and an adjacent linked product name that go to the same product page) this results in additional navigation and repetition for keyboard and screen reader users.
How to Fix It
If possible, combine the redundant links into one link and remove any redundant text or alternative text (for example, if a product image and product name are in the same link, the image can usually be given alt="").
The Algorithm... in English
Two adjacent links go to the same URL.
Standards and Guidelines
2.4.4 Link Purpose (In Context) (Level A)
Icon index

**Comment by dplaton** [comment]
Fixed by #548
